1.preprocessing_data/scripts/1.evaluate_qc.py

```diff
@@ -56,7 +56,7 @@
 channels = ["Actin", "DNA", "ER", "PM", "Mito"]
 
 # Create DataFrames for each channel with all Metadata columns
-Actin_df = qc_df.filter(like="Metadata_").copy()
+Actin_df = qc_df.filter(like="Metadata_").drop(columns=["Metadata_Series", "Metadata_Frame"]).copy()
```

Comment:
Why drop Series and Frame?

Reply:
Consider adding documentation for why?

Reply:
Great question! This is a CellProfiler thing, where it will automatically try to find "Frame" and "Series" metadata even if you do not have that in your data. These columns have all NaNs because we do not extract these and I remove them so it is easier to see the relevant metadata when performing QC. 

I can add documentation for this! Thanks for bringing it up since I have become so used to this output.